packages/cache/src/utils/hashKey.ts

```diff
@@ -0,0 +1,14 @@
+import type { Key } from '../types'
+import { type PlainObject, isPlainObject } from './isPlainObject'
+
+export const hashKey = (key: Key) =>
```

Comment:
```suggestion
export const hashKey = (key: CacheKey) =>
```

Reply:
I modified it https://github.com/toss/suspensive/commit/ceb14e435a346cfb2f3a4ea546dc60bb7444a34a